### DISCUSSION TITLE
Meaning of Bruggeman constant in Parameter Set

### DISCUSSION THREAD

**Discussion by fsenoner** [pybamm-team/PyBaMM#3831] [Q&A]
Dear PyBaMM Team,

I have a query regarding the Bruggeman constant parameter within the PyBaMM framework.

Could you please clarify the distinction between the two Bruggeman parameters listed below?

"Negative electrode Bruggeman coefficient (electrolyte)"
"Negative electrode Bruggeman coefficient (electrode)"

My understanding is that both parameters influence the effective diffusivity and conductivity. However, I'm uncertain whether one primarily affects the diffusivity in the solid phase while the other influences the diffusivity in the liquid phase and if both parameters are needed. There are a few parameter sets in pybamm where one of these two is set to zero.

The same question applies to the positive electrode parameters.

In the paper detailing the Chen2020 parameter set https://iopscience.iop.org/article/10.1149/1945-7111/ab9050/pdf, on page 18, it is mentioned:
"The Bruggeman coefficients are set to the theoretical value of 1.5 for packed spheres, given that the P2D model assumes this geometry for the electrodes. Setting the values to those in Table VII provides unphysical simulation results in which the electrolyte depletes after a few minutes of a 1C discharge."

Could you provide insights into why setting the Bruggeman coefficients to values other than those in Table VII leads to such unphysical simulation outcomes?

Thank you for your assistance.
Fabian

**Comment by TomTranter** [comment]
Hi Fabian,

Thanks for the question. Unfortunately it's not a simple straight-forward one to answer and the topic of tortuosity is quite tricky as calculated values depend on the type of transport being modelled and also the assumptions and boundary conditions applied in models that represent it. There are plenty of good reviews though and the work of Sam Coopers group in the battery field is a good place to start to read more.

"Negative electrode Bruggeman coefficient (electrolyte)" and "Negative electrode Bruggeman coefficient (electrode)" refer to transport properties of the negative electrode for the pore space (electrolyte filled region) and the solid space. i.e. transport through the different phases in the same volume averaged space. 

Both parameters are needed and the coefficients used in relations like Bruggeman don't necessarily have to be the same for each phase which is why the distinction is made. The coefficient b in eq 3 of the paper you refer to can be calculated in different ways and Bruggeman is only one example of a functional relationship between porosity and tortuosity. The method in the paper to determine the numbers in Table VII uses an image analysis technique - TauFactor which gives values that apparently when applied to the model does not give good agreement with observations. The 1C discharge is not possible due to electrolyte depletion - this means that the concentration of ions in the electrolyte reaches a value that is too low to provide any conductive pathway and the overpotential associated with electrolyte transport goes up to infinity and simulation fails. This situation happens if transport is restricted by having a tortuosity that is too high which is what the higher values of b represent. The authors state that the theoretical values of 1.5 are used and this gives better results and say that because the Bruggeman relations with values of 1.5 correspond to a sphere packing and the DFN also represents the particles as spheres this is a more physical representation.

In other words the value of Tortuosity of the battery as calculated by Taufactor which solves equations directly on images of the battery structures produced a value that was incompatible with an equation that is really only valid for a pack of spheres because battery electrodes are not packs of spheres (especially the anode). So the authors decided to change the coefficient to be one that was valid for a pack of spheres because the model also assumes that things are spherical and ordered. 2 wrongs make a right :) 

There is an open PR on transport efficiency which will allow for other functional forms and also direct specification of the transport efficiency with interpolated data
